build/common.project.props

```diff
@@ -14,19 +14,19 @@
     <NETFXTargetFrameworkVersion>v4.7.2</NETFXTargetFrameworkVersion>
     <NETFXTargetFramework>net472</NETFXTargetFramework>
     <NETCoreTargetFramework>netcoreapp3.1</NETCoreTargetFramework>
-    <NETCoreTargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net6.0</NETCoreTargetFramework>
-    <NETCoreTestTargetFramework>net6.0</NETCoreTestTargetFramework>
+    <NETCoreTargetFramework Condition="'$(DotNetBuildFromSource)' == 'true'">net7.0</NETCoreTargetFramework>
+    <NETCoreTestTargetFramework>net7.0</NETCoreTestTargetFramework>
```

Comment:
Please don't change the target framework in non source build scenarios. 
This can cause our builds to fail.